src/imp/openssl.rs

```diff
@@ -202,6 +202,9 @@ impl TlsConnector {
 
 /// OpenSSL-specific extensions to `TlsConnectorBuilder`.
 pub trait TlsConnectorBuilderExt {
+    /// Initialize `TlsAcceptorBuilderExt` from an `SslAcceptorBuilder`.
+    fn create(SslConnectorBuilder) -> Self;
```

Comment:
Can you use named parameters in the trait definitions? Can we call this `from_openssl` or something like that?